scripts/reserved-names/deny-labels.ts

```
@@ -0,0 +1,98 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import fs from 'fs';
+import { Transaction } from '@iota/iota-sdk/transactions';
+
+const LABEL_REGEX = /(?!-)[a-z0-9-]{0,62}[a-z0-9]/;
+
+// Parses a text file with comma-separated labels
+export const parseTextFile = (filePath: string): string[] => {
+    const fileContent = fs.readFileSync(filePath, 'utf8').trim();
+
+    if (!fileContent) {
+        return [];
+    }
+
+    return fileContent
+        .split(',')
+        .map((label) => label.trim())
+        .filter((label) => {
+            if (label.length === 0) {
+                return false;
+            }
+            if (!LABEL_REGEX.test(label)) {
+                throw new Error(`Invalid label provided: "${label}"`);
+            }
```

shouldn't we just remove the length check and let it fail here?

---

Would make it fail if it ends with a comma, but probably fine

---

Removed https://github.com/iotaledger/iota-names/pull/332/commits/d94aed049e1f80d12252306d2f32cedfda979cd8